builtin/principals/ldap/Principals.go

```diff
@@ -133,7 +135,7 @@ func (p Principals) Get(ctx context.Context, payload []byte) (context.Context, [
 	}
 
 	if len(gsr.Entries) == 0 {
-		return ctx, []string{}, errors.New("no group found for this user")
+		return ctx, []string{}, princsPkg.NewNotFoundError("ldap", "No group found for user "+ldapPrinc.User)
```

Comment:
We don't need to add ldapPrinc.User to error, the logger already add it as a field.